update-submodule/README.md

```diff
@@ -0,0 +1,48 @@
+# GitHub Action: Creates Pull Request when Submodules are Updated
+
+This GitHub action creates a new branch and pull request against the parent repository when submodules are updated.
+
+
+```yml
+
+---
+name: Submodule update
+
+on:
+  push:
+    branches: 
+    - master
+    - main
+    - "2.10"
+
+jobs:
+  build:
+    name: Submodule update
+    runs-on: docker
+    env:
+      PARENT_REPOSITORY: 'org/repo'
+      CHECKOUT_BRANCH: 'main'
+      PR_AGAINST_BRANCH: 'main'
+      PARENT_UPDATED_BRANCH: 'branch-for-update-'
```

Comment:
```suggestion
      PARENT_UPDATED_BRANCH: 'update-submodule-name'
```

Reply:
It is only example. Doesn't matter how shown in readme.